src/network/networkd-network.h

```diff
@@ -200,10 +200,14 @@ struct Network {
         /* CAN support */
         uint64_t can_bitrate;
         unsigned can_sample_point;
+        uint64_t can_data_bitrate;
```

Comment:
this is a 32bit value iiuc in `struct can_bittiming`, no? It should be here, too then.

Reply:
I changed `can_{data,}bitrate` to uint32_t to match the kernel setting.